Library/ENV/scm/git

```diff
@@ -10,8 +10,8 @@ fi
 exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
 #!/usr/bin/env ruby -W0
 
-# This script because we support $GIT, $HOMEBREW_SVN, etc. and Xcode-only
-# configurations. Order is careful to be what the user would want.
+# This script because we support $GIT, $HOMEBREW_SVN, etc., Xcode-only and
```

Comment:
I'm :+1: on not supporting a plain `$GIT` but worry a little about changing it after so long given `$GIT` is manpage-documented whereas `$HOMEBREW_GIT` isn't.